bookshelf.js

```diff
@@ -0,0 +1,64 @@
+
+//1-In our bookshelf.js file create a button using the createElement method.
+
+var btn = document.createElement("button");
+document.body.appendChild(btn);
+
+//2-Set the button's id attribute to fetch-books-btn using the setAttribute method.
+
+btn.setAttribute("id", "fetch-books-btn");
+
+//3-Create a text node with the text Fetch Books using the createTextNode method.
+
+var newtext = document.createTextNode("fetchBooks");
+
+//4-Append the text node to the button using the appendChild method.   
+
+btn.appendChild(newtext);
+
+//5-Add an event listener that will call a method called 
+//fetchBooks when the button is clicked using the addEventListener method.
+
+btn.addEventListener('click', fetchBooks);   
+
+                                       // Wiring up the button
+
+//1-Create a new function named fetchBooks
+
+// function fetchBooks(){
+// }
+// console.log ("hello")
+
+                                     //Fetching the books data
+
+                                     //1-Create a new function named fetchBooks
+
+function fetchBooks(){
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+    fetch(booksJSON)
+        .then(response => response.json())
+        .then(books => {
+            /* TODO: create the bookshelf */
+            console.log(books)
+        })  
+}
+
+                               //Creating the bookshelf
+ 
+
+function processBooks(booksJSON){
+    fetch(booksJSON)
```

Comment:
The code in this function should be in the fetchBooks function (if you look, you will see it already is, but instead of console.log on line 42, it should do `.then(processBooks)`. Then you need to write the code for this `processBooks` function per instructions